src/js/clickable-component.js

```diff
@@ -28,6 +28,7 @@ class ClickableComponent extends Component {
     this.on('click', this.handleClick);
     this.on('focus', this.handleFocus);
     this.on('blur', this.handleBlur);
+    player.on('fullscreenchange', this.handleFullscreenChange);
```

Comment:
why not do this only in `fullscreen-toggle`? Generic clickable-components don't need to know about it.